### ISSUE TITLE
Build Fails When pipeline_template key Not followed with New Line

### ISSUE THREAD

**Issue by brianwyka** [jenkinsci/config-driven-pipeline-plugin#26]
When there is a "\n" (new line) missing after `pipeline_template` in configuration file, build fails immediately.

```txt
Obtained pipeline.yml

GitHub has been notified of this commit’s build result

java.lang.StringIndexOutOfBoundsException: String index out of range: -18
	at java.lang.String.substring(String.java:1967)
	at org.jenkinsci.plugins.workflow.multibranch.template.finder.ConfigurationValueFinder.findFirstConfigurationValue(ConfigurationValueFinder.java:35)
	at org.jenkinsci.plugins.workflow.multibranch.template.ConfigFileSCMBinder.create(ConfigFileSCMBinder.java:169)
	at org.jenkinsci.plugins.workflow.job.WorkflowRun.run(WorkflowRun.java:293)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
Finished: FAILURE
```

## Works
```yaml
pipeline_template: maven


```

## Does Not Work
```yaml
pipeline_template: maven
```

**Comment by justinharringa** [comment]
Thanks for opening this @brianwyka! Opened up #28 with a fix. Once I've got the PR approved we should be able to merge and release.

Hope the plugin is working well for you and that you are doing well! Take care.

**Comment by justinharringa** [comment]
@brianwyka thanks again for your contribution! The fix for this has just been released to [version 1.3](https://github.com/jenkinsci/config-driven-pipeline-plugin/releases/tag/config-driven-pipeline-1.3). Have a great day!
